tests/data/latex-file.tex

```diff
@@ -0,0 +1,10 @@
+\documentclass[a4paper]{article}
+
+\begin{document}
+
+\LARGE
+
+Test LaTeX content
+
+\end{document}
+
```

Comment:
Sorting the tests into folders might be more sensible, since test data can normally not be shared;

```
tests/
  zathura-latex/
    latex-file.tex
    default.nix
```

Once there are hundreds of tests, this will help with naming files.

Reply:
Yes, sounds good.

Reply:
Actually, maybe having this test as a combined test was a mistake anyway.

Reply:
Separated tests for generation and viewing of PDFs, created some folders